bin/redbot_cli

```diff
@@ -45,7 +46,7 @@ def main() -> None:
     args = parser.parse_args()
 
     config_parser = ConfigParser()
-    config_parser.read_dict({"redbot": {"lang": "en", "charset": "utf-8"}})
+    config_parser.read(os.environ.get("REDBOT_CONFIG", "config.txt"))
```

Comment:
I suspect it'd be better to use `$XDG_CONFIG_HOME`, defaulting to `$HOME/.config` as per [the spec](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html).

Something like:

```suggestion
    home_dir = os.environ.get("HOME")
    config_dir = os.environ.get("XDG_CONFIG_HOME", f"{home_dir}/.config")
    config_loc = f"{config_dir}/redbot.cfg"
    if os.path.exists(config_loc):
        config_parser.read(config_loc)
    else:
        config_parser.read_dict({"redbot": {"lang": "en", "charset": "utf-8"}})
```

(not tested)

Reply:
I copied how to read the config file from https://github.com/mnot/redbot/blob/main/bin/redbot_cgi.py#L82
I guess both ways are fine, but it is probably least confusing if all files use the same mechanism to read the config file.

Reply:
`redbot_cgi.py` is no more, so that shouldn't be an issue any more.